packages/cli/lib/base-generator.js

```diff
@@ -292,14 +313,17 @@ module.exports = class BaseGenerator extends Generator {
   /**
    * Print out the exit reason if this generator is told to exit before it ends
    */
-  end() {
+  async end() {
     if (this.shouldExit()) {
       debug(this.exitGeneration);
       this.log(chalk.red('Generation is aborted:', this.exitGeneration));
       // Fail the process
       process.exitCode = 1;
-      return false;
+      return;
+    }
+    if (this.options.format) {
+      this.log('Running npm run lint:fix to format the code...');
+      await this._runNpmScript(this.destinationRoot(), ['run', 'lint:fix']);
```

Comment:
Should there be a check to see if the script exists? Or does `_runNpmScript` do that under the hood?

Reply:
If the script does not exist, an error will be reported.